kibana-reports/server/routes/utils/constants.ts

```diff
@@ -66,3 +66,5 @@ export enum TRIGGER_TYPE {
 }
 // https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-request-from-size.html
 export const DEFAULT_MAX_SIZE = 10000;
+
+export const LOCAL_HOST = 'http://localhost:5601';
```

Comment:
Should we be hard-coding the host like this?

Reply:
Ideally, we should extract the server host/port from `kibana.yml`, but the new platform applies some restrictions to access certain fields. More details can be found in this issue #98

Reply:
Can we get the local host from a `window.location` or `document` method instead? I think those should be fine to access in the new platform.

Reply:
That `window.location` won't work on server side. The UI side still keeps the usage of `window.location` which is fine.

Reply:
Ack, miss from me- forgot this was on server-side